README.md

```diff
@@ -0,0 +1,38 @@
+# Ahoy::Intercom
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'ahoy-intercom'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install ahoy-intercom
+
+## Usage
+
+Set your environment variable `INTERCOM_ACCESS_TOKEN`.
+About access token see [here](https://developers.intercom.com/docs/personal-access-tokens)
+
+Inherit your Ahoy::Store fom Ahoy::Intercom::Store in `/config/initializers/ahoy.rb`
```

Comment:
Why? In Ahoy docs all stores are installed like "rails generate ahoy:stores:mongoid"
"